src/frontend/screens/Settings/components/AutoDXVK.tsx

```diff
@@ -17,7 +17,7 @@ const AutoDXVK = () => {
   const { appName } = useContext(SettingsContext)
   const [installingDxvk, setInstallingDxvk] = React.useState(false)
 
-  if (wineVersion.type !== 'wine') {
+  if (wineVersion.type !== 'wine' || wineVersion.bin.includes('toolkit')) {
```

Comment:
```suggestion
  if (wineVersion.type !== 'wine' || wineVersion.type === 'toolkit') {
```

Reply:
I am actually treating the wine from toolkit as wine since we are using the binary. And I have removed the support for the apple scripts since they are basically useless.
But since if we install DXVK on a GPTK wine prefix it breaks the prefix, the check needs to be done this way.